test/verify/check-login

```diff
@@ -48,6 +48,14 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
         m.start_cockpit()
         b.open("/system")
 
+        def assert_no_login():
+            users = m.execute("who | grep web | cat") # |cat for true return status
+            assert users == ""
+
+        def assert_login(user):
+            users = m.execute("who | grep '^" + user + " *web'")
+            assert users != ""
```

Comment:
Likewise, something like `self.assertRegex(m.execute("who"), "^%s\b.*web console" % user)`?

Also, as this isn't fixed in the RHEL 8.1 version, let's put the special-casing here. Something like
```py
# Fixed in PR #13242 
if m.image in ["rhel-8-1-distropkg"]:
    return
```